microservices/payment/src/entities/transaction.ts

```diff
@@ -1,6 +1,74 @@
-import { Entity } from 'typeorm';
+import { IsUndefinable } from '@lomray/microservice-helpers';
+import { Allow } from 'class-validator';
+import { JSONSchema } from 'class-validator-jsonschema';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryColumn } from 'typeorm';
+import Customer from '@entities/customer';
 
+@JSONSchema({
+  properties: {
+    customer: { $ref: '#/definitions/Customer' },
+  },
+})
 @Entity()
-class Transaction {}
+class Transaction {
+  @PrimaryColumn({ type: 'varchar', length: 66 })
+  @Allow()
+  id: string;
+
+  @Column({ type: 'varchar', length: 100 })
+  @IsUndefinable()
+  title: string;
+
+  @Column({ type: 'varchar', length: 18 })
+  @Allow()
+  customerId: string;
+
+  @Column({ type: 'varchar', length: 66 })
+  @IsUndefinable()
+  bankAccountId: string;
+
+  @Column({ type: 'varchar', length: 66 })
+  @IsUndefinable()
+  cardId: string;
+
+  @Column({ type: 'varchar', length: 32 })
+  @IsUndefinable()
+  entityId: string;
+
+  @Column({ type: 'int' })
+  @Allow()
+  amount: number;
```

Comment:
it's can be e.g. 12.45 also, so int not correct

Reply:
Stripe uses integers for prices like 12.5 $ is equal 1250 (cents). I checked paypal, paypal uses even strings. Can I leave integers for now?

Reply:
![image](https://user-images.githubusercontent.com/48718981/235444008-6f7cd129-de71-439c-a68a-c82cd5304970.png)

Reply:
ok, for all integrations will be used as int